components/shopify/actions/add-tags/add-tags.mjs

```diff
@@ -0,0 +1,83 @@
+import shopify from "../../shopify.app.mjs";
+
+export default {
+  name: "Add Tags",
+  version: "0.0.1",
+  key: "shopify-add-tags",
+  description:
+    "Add tags. [See the docs](https://shopify.dev/api/admin-graphql/2022-07/mutations/tagsadd)",
+  props: {
+    shopify,
+    resource: {
+      label: "Resource Type",
+      type: "string",
+      description: "The Shopify Admin API resource type.",
+      options: [
+        "Product",
+        "Customer",
+        "Order",
+        "DraftOrder",
+        "OnlineStoreArticle",
+      ],
+    },
+    id: {
+      label: "Resource ID",
+      type: "string",
+      description:
+        "The Shopify Admin Resource ID. For example, the ID of a Product, Customer, Order, DraftOrder, or OnlineStoreArticle. Can be found at the end of the URL in Shopify Admin.",
+    },
+    tags: {
+      label: "Tags",
+      type: "string",
+      description: "Separate tags with a comma to add multiple tags.",
+    },
+  },
+  type: "action",
+  async run({ $ }) {
+    const {
+      resource,
+      id,
+    } = this;
+    const gid = `gid://shopify/${resource}/${id}`;
+    let tags = [
+      this.tags,
+    ];
+
+    if (tags.includes(",")) {
+      tags = tags.split(",").map((item) => item.trim());
+    }
+
+    const mutation = `
+      mutation tagsAdd($id: ID!, $tags: [String!]!) {
+        tagsAdd(id: $id, tags: $tags) {
+          node {
+            id
+          }
+          userErrors {
+            field
+            message
+          }
+        }
+      }
+    `;
+
+    const variables = {
+      id: gid,
+      tags: tags,
+    };
+
+    const res = await this.shopify
+      .getShopifyInstance()
+      .graphql(mutation, variables);
```

Comment:
If possible, can you please use [_makeGraphQlRequest](https://github.com/PipedreamHQ/pipedream/blob/495e324420b86b831e438521dac8028473db6241/components/shopify/shopify.app.mjs#L433) after moving the call to the [app file](https://github.com/PipedreamHQ/pipedream/blob/master/components/shopify/shopify.app.mjs)?

Reply:
@feyzullah With the move to the app file, I am now using `_makeGraphQlRequest`. Thanks for pointing me to that function didn't realize it was already there. 🙂